File: practice/data_structures/trees/postorder_traversal/postorder_traversal.py
"""
Node is defined as
self.left (the left child of the node)
self.right (the right child of the node)
self.data (the value of the node)
"""

def postOrder(i_root):
    if i_root:
        postOrder(i_root.left)
        postOrder(i_root.right)
        print i_root.data,

File: practice/data_structures/trees/postorder_traversal/postorder_traversal.cpp
/* you only have to complete the function given below.  
Node is defined as  

struct node
{
    int data;
    node* left;
    node* right;
};

*/


void postOrder(node * i_root)
{
    if (i_root) {
        postOrder(i_root->left);
        postOrder(i_root->right);
        cout << i_root->data << " ";
    }
}

